artiq/gateware/eem_7series.py

```diff
@@ -110,6 +110,13 @@ def peripheral_phaser(module, peripheral, **kwargs):
     eem.Phaser.add_std(module, peripheral["ports"][0], **kwargs)
 
 
+def peripheral_hvamp(module, peripheral, **kwargs):
+    if len(peripheral["ports"]) != 1:
+        raise ValueError("wrong number of ports")
+    eem.HVAmp.add_std(module, peripheral["ports"][0], 
+        ttl_serdes_7series.Output_8X, **kwargs)
```

Comment:
Do we really need 1ns resolution on these?

Reply:
Not really, changing to `ttl_simple`